app.py

```diff
@@ -206,7 +231,9 @@ def before_request_https():
     app.config['SECRET_KEY'] = os.getenv('SECRET_KEY')
 
 else:
-    app.config['SECRET_KEY'] = os.getenv('SECRET_KEY', uuid.uuid4().hex)
+    # The value doesn't matter for dev environments, but it needs to be a constant
+    # so that our cookies don't get invalidated every time we restart the server.
+    app.config['SECRET_KEY'] = os.getenv('SECRET_KEY', 'WeAreDeveloping')
```

Comment:
I propose to change the key `WeAreDeveloping` to `WeAreInventing`, honoring the brilliance of Hedy Lamarr